flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/operators/transform/TransformDataOperatorTest.java

```diff
@@ -555,6 +574,64 @@ void testTimestampDiffTransform() throws Exception {
                 .isEqualTo(new StreamRecord<>(insertEventExpect));
     }
 
+    @Test
+    void testNullCastTransform() throws Exception {
```

Comment:
Great to see this! Could you please insert more non-null data row to test 1) non-null value casting and verify results and 2) unsupported type conversion failure?

Reply:
@yuxiqian Looks like @aiwenmo is busy on other things, as this comment is a minor test-related improvement and this PR block some on-going PRs, I'll merge this PR soon.